demo/locales/source.json

```diff
@@ -303,6 +303,10 @@
     "description": "The name of a configuration value.",
     "message": "Ignore [PROPER_NAME:HLS] Text Stream Failures"
   },
+  "DEMO_FORCE_HLS_FULL_SEGMENT_REQUEST": {
+    "description": "The name of a configuration value.",
+    "message": "Force Full Segment Request"
```

Comment:
Can you please update the constant names and the messages, as well?  The config name and its description in externs/ are good, but the constants and the translations should match that.

Reply:
done